### PR TITLE
Add missing nullability annotations to some Forge-added methods

### PR DESCRIPTION
**Pull request by bs2609**
Fairly self-explanatory cleanup PR, just adding `@Nullable` annotations to some methods that are patched in by Forge.

### CODE REVIEW

File: patches/minecraft/net/minecraft/tileentity/TileEntityChest.java.patch [hunk 1]
```diff
@@ -28,6 +29,7 @@
 +        return super.getCapability(capability, facing);
 +    }
 +
++    @Nullable
```
Comment (mezz) [inline patches/minecraft/net/minecraft/tileentity/TileEntityChest.java.patch]: Can this one ever actually return null?

**Review by mezz — CHANGES_REQUESTED** [review]

File: patches/minecraft/net/minecraft/tileentity/TileEntityChest.java.patch (hunk 1 above)
Comment (bs2609) [reply patches/minecraft/net/minecraft/tileentity/TileEntityChest.java.patch]: Looking at it more closely, it certainly seems it's not supposed to.
I'll remove the annotation here.

**Comment by bs2609** [comment]
Also, do the `getBlock`/`getBlockRaw` methods for `ItemBlock`/`ItemBlockSpecial` need annotating `@Nullable`? I'm not sure if the underlying field is intended to be nullable or not.

**Comment by mezz** [comment]
I think those two should be left as-is. They're similar to `ItemStack.getItem()` and `ItemStack.getItemRaw()` which are basically nonnull except in certain exceptional/internal cases.

**Review by mezz — APPROVED** [review]

**Comment by LexManos** [comment]
Honestly wouldn't bother with get*Raw functions as they are purely private methods to bypass our transformers.

**Review by liach — APPROVED** [review]

**Pull request merged**